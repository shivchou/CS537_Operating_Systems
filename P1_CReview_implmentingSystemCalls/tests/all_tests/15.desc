PART 2: Call user program pwd from subdirectory.
