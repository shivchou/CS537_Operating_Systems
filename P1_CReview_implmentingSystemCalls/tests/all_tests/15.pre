./edit-makefile.sh ../solution/part2/Makefile test_7 > ../solution/part2/Makefile.test
cp -f tests/test_7.c ../solution/part2/test_7.c
cd ../solution/part2/
make clean
make -f Makefile.test clean
make -f Makefile.test xv6.img
make -f Makefile.test fs.img
cd ../../tests
