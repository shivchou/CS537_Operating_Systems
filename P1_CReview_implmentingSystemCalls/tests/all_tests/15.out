XV6_TEST_OUTPUT before making directory and changing to it.
XV6_TEST_OUTPUT calling pwd from subdirectory:foobar
